tools/wave/network/api/api_handler.py

```diff
@@ -5,13 +5,16 @@
 import traceback
 import logging
 
-from urllib.parse import parse_qsl
+try:
+    from urllib.parse import parse_qsl
+except ImportError:
+    from urlparse import parse_qsl
```

Comment:
Do you need this? It's throwing 2 errors:

> error: Cannot find implementation or library stub for module named "urlparse" [import-not-found]
> error: Name "parse_qsl" already defined (possibly by an import) [no-redef]

and other parts of the project use `urllib.parse` and [`parse_qsl` is in python 3.8](https://docs.python.org/3.8/library/urllib.parse.html#urllib.parse.parse_qsl), so I assume that's fine?